app/views/hq/orphans/index.html.erb

```diff
@@ -90,6 +133,21 @@
             <td>
               <%= orphan.sponsorship_status.humanize %>
             </td>
+            <td>
+              <%- if params[:sponsor_id] and params[:scope] == 'eligible_for_sponsorship' %>
+                  <form action=<%= "/hq/sponsors/#{@sponsor.id}/sponsorships" %> method="post">
+                    <input name="_method" type="hidden" value="post">
+                    <input name="utf8" type="hidden" value="&#x2713;" />
+                    <input name="authenticity_token" type="hidden" value="<%= form_authenticity_token %>" />
+                    <div class="form-group">
+                      <label for="title">Beginning</label>
+                      <input name="sponsorship_start_date" type="text" value="<%= Date.current.to_s %>" class="form-control date-picker">
+                      <input name="orphan_id" type="hidden" value="<%= orphan.id %>" />
+                      <input type="submit" value="Sponsor this orphan" class="btn btn-primary" />
+                    </div>
+                  </form>
```

Comment:
why not use `form_tag` instead of plain html?
I think `action` should be `hq_sponsor_sponsorships(@sponsor)`. check the `rake routes` to be sure